templates/layout.jade

```diff
@@ -5,6 +5,7 @@ html
 		link(rel='stylesheet', href='/stylesheets/font-awesome.css')
 		link(rel='stylesheet', href='/stylesheets/style.css')
 		link(rel='stylesheet', href='/stylesheets/lib/select2.css')
+		link(rel='stylesheet', href='/stylesheets/lib/dndSelector.css')
```

Comment:
It was added here, https://github.com/bookbrainz/bookbrainz-site/pull/101/commits/627e6c339064402421958bf1fbb4d75cbb544d1c
I think git refused to add it for a while because of the gitignore.